language/Portuguese_Brazil/strings.po

```diff
@@ -1992,19 +2001,19 @@ msgstr "Aumentar as imagens no painel direito"
 
 #: src/frontend/mame/ui/submenu.cpp:26
 msgid "Show mouse pointer"
-msgstr "Mostrar ponteiro do mouse"
+msgstr "Mostrar o ponteiro do mouse"
 
 #: src/frontend/mame/ui/submenu.cpp:27
 msgid "Confirm quit from machines"
-msgstr "Confirmar saída das máquinas"
+msgstr "Pedir confirmação ao sair"
```

Comment:
This makes terminology inconsistent - "machines" in the sense of emulated systems is being translated as "máquinas" elsewhere.